components/org.wso2.carbon.identity.application.authenticator.fido2.endpoint/src/main/java/org/wso2/carbon/identity/application/authenticator/fido2/endpoint/common/Constants.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.identity.application.authenticator.fido2.endpoint.common;
+
+/**
+ * Common constants for FIDO2 API.
+ */
+public class Constants {
+
+    public static final String EQUAL_OPERATOR = "=";
+    private static final String FIDO2_ERROR_CODE_PREFIX = "FID-";
+
+    /**
+     * Enum for error messages.
+     */
+    public enum ErrorMessages {
+
+        ERROR_CODE_START_REGISTRATION("50001", "Error starting the FIDO2 device registration flow.",
+                "A system error occurred while serializing start registration response for the appid : %s"),
+        ERROR_CODE_FINISH_REGISTRATION("50002", "Error finishing FIDO2 device registration process.",
+                "A system error occurred while finishing device registration."),
+        ERROR_CODE_START_REGISTRATION_INVALID_ORIGIN("50003", "Error starting the FIDO2 device " +
+                "registration flow.", "FIDO2 trusted origin: %s sent in the request is invalid."),
+        ERROR_CODE_FETCH_CREDENTIALS("50004", "Error while retrieving user credentials.",
+                "A system error occurred while retrieving user credentials " +
+                        "for the user : %s"),
```

Comment:
Will it exceed the column length if line 39 and 40 are merged?

Reply:
Fixed it into 1 line.